server/views/pages/driftRadiator.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% set pageTitle = applicationName + " - " + component.name %}
+{% set mainClasses = "app-container govuk-body" %}
+{% set devEnvName = component.environments[0].name %}
+
+{% block content %}
+
+{{ govukBackLink({
+  text: "Back",
+  href: "/components/" + component.name
+}) }}
+
+<h1 id="detailPageTitle">Deployment Drift Radiator</h1>  
+<h2>{{ component.name }}</h2>  
+    <p><a class="govuk-link--no-visited-state" href="https://app.circleci.com/pipelines/github/ministryofjustice/{{ component.name}}?branch=main">Circle CI pipeline</a></p>
```

Comment:
Assumes all services are on circle - I was hoping to link through to builds on each tile but as well as build number, also need workflow id, which isn't available

Reply:
the old monitor called the circleci API to get the latest build status etc.

Reply:
it's a shame we don't have the pipeline id available as part of the build number (as well as the job number) 

Then we could link to this page:
https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-developer-portal/544

Reply:
there's [an api](https://circleci.com/docs/api/v2/index.html#operation/getJobDetails) that grabs the pipeline id for job.

You could then grab the pipeline and build up any url.

Which would support:
"As a user, I want to easily progress a known deployment from one environment to the next"